PiwikTracker/Tracker.swift

```diff
@@ -143,7 +143,31 @@ extension Tracker {
             actionName: action,
             language: Locale.httpAcceptLanguage,
             isNewSession: false, // set this to true once we can start a new session
-            referer: nil)
+            referer: nil,
+            eventCategory: nil,
+            eventAction: nil,
+            eventName: nil,
+            eventValue: nil
+        )
+    }
+    internal func event(withCategory category: String, action: String, name: String? = nil, value: Float? = nil) -> Event {
```

Comment:
Here the `action` parameter is used for `eventAction`, but in `event(action: [String])` it is used for actionName. I find this irritating. What about using `action` for `actionName` and `eventAction` for `eventAction` consistantly?